frontend/lib/uast/Record.cpp

```diff
@@ -31,9 +31,12 @@ owned<Record> Record::build(Builder* builder, Location loc,
                             Decl::Linkage linkage,
                             owned<AstNode> linkageName,
                             UniqueString name,
+                            AstList interfaceExprs,
```

Comment:
(this comment is not about this line at all)

Could you implement `std::string Record::dumpChildLabelInner(int i) const` so that we get "implements interface" next to the relevant uAST nodes in dumps?